config/jobs/kubernetes/sig-release/release-branch-jobs/1.21.yaml

```diff
@@ -288,7 +288,7 @@ periodics:
     org: kubernetes
     path_alias: k8s.io/kubernetes
     repo: kubernetes
-  - base_ref: master
+  - base_ref: release-1.21
```

Comment:
This problem is related to: https://github.com/kubernetes/test-infra/issues/17097

Can we not just fix the actual jobs, but address that bug too to avoid those issues in the future?

Reply:
I believe @jupblb is looking into it currently.

Reply:
I've opened PR for fixing forker: #22984